Makefile

```diff
@@ -35,18 +35,14 @@ config-devnet:
 	PATH="./target/release:$(PATH)" \
 	./scripts/setup.sh
 
-run-devnet-iroh:
-	cargo install iroh-cli --version 0.28.1
-	iroh --rpc-addr 127.0.0.1:4919 start
-
 run-devnet-fendermint:
 	rm -rf ~/.fendermint/data/rocksdb
 	FM_NETWORK=test \
 	FM_TRACING__CONSOLE__LEVEL=info,fendermint=debug,recall_executor=debug \
 	FM_VALIDATOR_KEY__PATH=keys/validator.sk \
 	FM_VALIDATOR_KEY__KIND=regular \
 	FM_RESOLVER__CONNECTION__LISTEN_ADDR=/ip4/127.0.0.1/tcp/3001 \
-	./target/release/fendermint run --iroh-addr 127.0.0.1:4919
+	./target/release/fendermint run --iroh-path iroh-resolver-path # TODO: what path
```

Comment:
not sure what path to use here, please advise

Reply:
for devnet, I think `.config/recall/data/iroh` makes sense. I guess we need two paths right? one for each iroh instance. `.config/recall/data/iroh-staging` and `.config/recall/data/iroh-chain` ?

for the 3-node localnet setup, i can help there... it's kind of a pain to deal with the `cargo make` scripts

Reply:
> for the 3-node localnet setup, i can help there... it's kind of a pain to deal with the cargo make scripts

I think I have that part figured out, but definitely needs review 😅